src/main/java/build/buildfarm/common/redis/RedisMap.java

```diff
@@ -35,13 +40,32 @@ public class RedisMap {
    */
   private final String name;
 
+  /**
+   * @field defaultExpiration_s
+   * @brief The expiration time to use on inserts when none is given.
+   * @details The map can be initialized with a default expiration. In doing so, expirations can be
+   *     omitted from calls to insert.
+   */
+  private final int defaultExpiration_s;
+
   /**
    * @brief Constructor.
    * @details Construct a named redis map with an established redis cluster.
    * @param name The global name of the map.
    */
   public RedisMap(String name) {
     this.name = name;
+    this.defaultExpiration_s = 86400;
+  }
+
+  /**
+   * @brief Constructor.
+   * @details Construct a named redis map with an established redis cluster.
+   * @param name The global name of the map.
+   */
+  public RedisMap(String name, int timeout_s) {
+    this.name = name;
+    this.defaultExpiration_s = timeout_s;
```

Comment:
nit: call the verbose constructor with the terse one and that 86400 default. And this isn't the 'default' expiration, it's the current one, worthy of just being called 'expiration.'

Reply:
fixed